src/components/molecules/MarketStats.tsx

```diff
@@ -62,20 +86,9 @@ export default function MarketStats(): ReactElement {
       Total of <strong>{stats?.datasets.total}</strong> data sets & unique
       datatokens published by <strong>{stats?.owners}</strong> accounts.
       <br />
-      <PriceUnit
-        price={`${stats?.ocean}`}
-        small
-        className={styles.total}
-        conversion
-      />{' '}
-      and{' '}
-      <PriceUnit
-        price={`${stats?.datatoken}`}
-        symbol="datatokens"
-        small
-        className={styles.total}
-      />{' '}
-      in <strong>{stats?.datasets.pools}</strong> data set pools.
+      <Conversion price={`${totalLockedValue}`} hideApproximationSign /> TVL
```

Comment:
we can make use of HTML as semantically intended for [abbreviations](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr) to get this sentence shorter:

```html
<abbr title="Total Value Locked">TVL</abbr>
```

Not sure we have any default `abbr` styles though so that might need a tweak